### DISCUSSION TITLE
Github profile

### DISCUSSION THREAD

**Discussion by jeemanifaster** [community/community#78739] [Other Feature Feedback, Questions, & Ideas] [answered]
I always see people with their fancy profiles GitHub, mine is only projects Ive pinned, how do people do it?

**Comment by walidbosso** [comment, marked as answer]
1. Create a new repository with your GitHub username (e.g., github.com/your-username/your-username).
2. Add a README.md file to this repository.
3. Customize the README with information about yourself, your skills, and any other details you want to showcase. You can use Markdown for formatting.
4. This same README file will Appear on ur profile
